controller/user_test.go

```diff
@@ -248,6 +256,13 @@ func (m TestUserRepository) Load(ctx context.Context, id uuid.UUID) (*account.Us
 	return m.User, nil
 }
 
+func (m TestUserRepository) Exists(ctx context.Context, id string) (bool, error) {
+	if m.User == nil {
+		return false, errors.New("not found")
```

Comment:
@kwk correct me if I am wrong but I am using `errs` which normally is the pkg  "github.com/pkg/errors"